src/lib/transition_frontier/full_frontier/full_frontier.ml

```diff
@@ -739,7 +741,9 @@ let apply_diffs t diffs ~enable_epoch_ledger_sync ~has_long_catchup_job =
     )
   in
   [%log' trace t.logger] "after applying diffs to full frontier" ;
-  if (not (enable_epoch_ledger_sync = `Disabled)) && not has_long_catchup_job
+  if
+    (match enable_epoch_ledger_sync with `Disabled -> false | _ -> true)
```

Comment:
Would also like to see this match pulled outside of the `if` for readability.

Reply:
Similarly here